cypress/e2e/test1/checkHeadline.cy.js

```diff
@@ -0,0 +1,8 @@
+/// <reference types="cypress" />
+
+describe('chelling2 tests', () => {
+    it('has appropriate headline', () => {
+        cy.visit('https://work.co/').get('[data-test-id="header-title-text"]').should('have.text', 'We design and ship digital products that transform companies.')
```

Comment:
This works, but try splitting it into 2 commands:
`cy.visit();` and `cy.get()...` (the second one in new line)
that would look much nicer. Also, update desc text from "chelling2 tests" to something describing the test more.
Overall, great work 👏  🔝